gadgets/Makefile

```diff
@@ -1,9 +1,11 @@
 SHELL := /bin/bash
 
-GADGET_TAG ?= $(shell ../tools/image-tag branch)
-GADGET_REPOSITORY ?= ghcr.io/inspektor-gadget/gadget
 BUILDER_IMAGE ?= ghcr.io/inspektor-gadget/ebpf-builder:latest
-IG ?= ig
+
+export GADGET_TAG ?= $(shell ../tools/image-tag branch)
+export GADGET_REPOSITORY ?= ghcr.io/inspektor-gadget/gadget
+export IG ?= ig
```

Comment:
> Do you mean the export keyword?
>
Yes, but I meant that they are not needed because we are not using those env variables inside the tests anymore. We are now passing the image name "trace_open" and letting `ig` add the default repo and tag (this way, we also test such a feature).

Reply:
However, for `IG` it's still needed, so I also vote for the second option as we are doing for `KUBECTL_GADGET`:

https://github.com/inspektor-gadget/inspektor-gadget/blob/abd4dfda26fec0def8735bc2f1de21b9c51c84c2/Makefile#L276-L288